Xiaomi/RedmiNote10/AndroidManifest.xml

```diff
@@ -0,0 +1,10 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+        package="me.phh.treble.overlay.xiaomi.redminote10"
+        android:versionCode="1"
+        android:versionName="1.0">
+        <overlay android:targetPackage="android"
+                android:requiredSystemPropertyName="ro.product.board"
```

Comment:
FYI. **`ro.product.board`** exists on stock props. I didn't had other common fingerprint attributes, because they get modified when we spoof fingerprint to fix safetynet issues. And overlay will be rendered useless.